Assets/Tests/InputSystem/APIVerificationTests.cs

```diff
@@ -412,9 +416,17 @@ private static bool IgnoreMethodForDocs(MethodDefinition method)
     }
 
     #if HAVE_DOCTOOLS_INSTALLED
-    ////TODO: move this to a fixture setup so that it runs *once* for all API checks in a test run
     private static string GenerateDocsDirectory(out string log)
     {
+        // only generate the docs once for any test run. This isn't in a one-time setup fixture method
```

Comment:
Question, would it make more sense to move out docs tests into separate test class and do this under `[OneTimeSetUp]`?

I'm not gonna be too annoying about this, but we use NUnit for a reason, otherwise every test would have some custom voodoo setup code around it.

PS. Fine merging it any way